styles/details.scss

```diff
@@ -0,0 +1,54 @@
+.details {
+    position: relative;
+    width: 1920px;
```

Comment:
Fixed width and height is not a good thing, width should be 100%, and we don't need to setup the height, it would adjust based on content inside